wpull/warc/recorder.py

```diff
@@ -31,21 +31,10 @@
 _ = gettext.gettext
 
 
-WARCRecorderParams = namedlist.namedtuple(
+WARCRecorderParams = namedtuple(
     'WARCRecorderParamsType',
-    [
-        ('compress', True),
-        ('extra_fields', None),
-        ('temp_dir', './'),
-        ('log', True),
-        ('appending', False),
-        ('digests', True),
-        ('cdx', None),
-        ('max_size', None),
-        ('move_to', None),
-        ('url_table', None),
-        ('software_string', None)
-    ]
+    ('compress','extra_fields','temp_dir','log','appending','digests','cdx','max_size','move_to','url_table','software_string'),
+    defaults=(True, None, './', True, False,True, None,None,None,None,None)
```

Comment:
I don't like how the defaults are specified for a namedtuple, especially the wide one here. Is there a better way to do this with [attrs](https://www.attrs.org/en/stable/) or [dataclasses](https://docs.python.org/3/library/dataclasses.html) or similar?